docs/References.bib

```diff
@@ -128,22 +128,22 @@ @article{Fishbone1976apj
 }
 
 @article{Foucart2015vpa,
```

Comment:
and what determines the formatting?  If I copy from inspire, I get the formatting you are changing...

Reply:
Consistency with the majority of the file, where keys are indented by two spaces, and the equal signs are aligned two spaces beyond the longest key.

Reply:
well to be honest, this is not something I'll be doing when adding references, so if there is a desire to unify the format there will need to be a tool to do so...

Reply:
@kidder I definitely agree, and we already have an issue for this: #1381.